src/VirtoCommerce.Platform.Data/Common/FileSystemCountriesService.cs

```diff
@@ -53,5 +58,10 @@ public async Task<IList<CountryRegion>> GetCountryRegionsAsync(string countryId)
 
             return Array.Empty<CountryRegion>();
         }
+
+        public Country GetByCode(string code)
+        {
+            throw new NotImplementedException();
```

Comment:
is there a reason we can't implement this method?

Reply:
We don't use this service in any way but getting countries regions. It is possible to implement this method, but do we really need this implementation?

Reply:
I think we should fully implement interfaces